packages/ckeditor5-engine/src/view/downcastwriter.js

```diff
@@ -191,7 +191,7 @@ export default class DowncastWriter {
 	createAttributeElement( name, attributes, options = {} ) {
 		const attributeElement = new AttributeElement( this.document, name, attributes );
 
-		if ( options.priority ) {
+		if ( options.priority !== undefined ) {
```

Comment:
```suggestion
		if ( typeof options.priority === 'number' ) {
```

Given that `options.priority` is documented as a number let's be a little more pedantic about what we take here. priority=true or priority='foo' doesn't make much sense here.